src/string_list.h

```diff
@@ -34,6 +34,9 @@ struct string_list *copy_string_list(const struct string_list *sl);
 struct string_list *sl_from_str(const char *string);
 struct string_list *sl_from_strn(const char *string, size_t len);
 
+// Return a string list with the given string consumed
```

Comment:
Can the comment say something like "taking ownership of the string"?  I think that would help add clarity for someone who's confused by the term "consume" in the function name.

Reply:
Updated the function comment to:
```
// Return a string list with the given string as single item
// Takes ownership of the given string
```